client/src/components/containers/CodeReviewMetricsContainers/CodeReviewFirstResponse/CodeReviewFirstResponse.module.scss

```diff
@@ -0,0 +1,18 @@
+@import "src/styles/index.scss";
+
+.firstReviewResponseTimeContainer {
+  @include flex-align-center;
+  @include flex-justify-space-between;
+
+  width: 100%;
+}
+
+.firstReviewResponseTime {
+  width: 90%;
```

Comment:
Use flex space between and gap to set spacing

Reply:
this i used when I have info icon, now we changed the icon, without width also it's working fine.